AUTHORS

```diff
@@ -0,0 +1,7 @@
+Cédric Jeanneret <cjeanner@redhat.com>
+Haikel Guemar <hguemar@fedoraproject.org>
+Jason Guiditta <jason.guiditta@gmail.com>
+Jason Joyce <fuzzball81@gmail.com>
+Jason Joyce <jjoyce@redhat.com>
```

Comment:
I am not sure we want both of Jason's emails here? Don't feel strongly, just seems kind of pointless to have both.

Reply:
Must be the result of an automatic process, wasn't sure if I should mess with it. There is already the AUTHORS.rst.